### ISSUE TITLE
Make sure all instance variables are initialized

### ISSUE THREAD

**Issue by ioquatix** [titusfortner/webdrivers#62]
```
lib/webdrivers/chromedriver.rb:42: warning: instance variable @downloads not initialized
```

**Comment by titusfortner** [comment]
Yeah, we need to refactor out a lot of this code since we are directly pulling from URL / file rather than parsing a list.

**Comment by titusfortner** [comment]
This should be fixed with 480388fdb21d5c14c8e4ac93b07ae8c0ee2d79f8
Let me know if there are still issues, and I can reopen
